i18n/en/docusaurus-plugin-content-docs/version-v1.0/user/registry/eureka.md

```diff
@@ -49,35 +49,32 @@ First, you need to add the Maven dependency of `spring-cloud-starter-netflix-eur
 
 ### Configuring the Registry on the Client Side
 
-Add the corresponding configuration to [**application.yml**](https://github.com/seata/seata/blob/develop/script/client/spring/application.yml), and refer to the [Configuration Reference](https://github.com/seata/seata/tree/develop/script/client) for other configurations.
+Add the corresponding configuration to [**application.yml**](https://github.com/seata/seata/blob/1.0.0/script/client/spring/application.yml), and refer to the [Configuration Reference](https://github.com/seata/seata/tree/1.0.0/script/client) for other configurations.
 
 ```yaml
 seata:
-  tx-service-group: my_test_tx_group
-  service:
-    vgroup-mapping:
-      my_test_tx_group: seata-server # Configure the corresponding value of registry.eureka.application on the Server side here
   registry:
     type: eureka
     eureka:
-      service-url: http://localhost:8761/eureka
+      application: default
```

Comment:
It needs to be clear where the eureka.application configuration is being used. It's not uniform here. Isn't that a problem?